requirements.txt

```diff
@@ -0,0 +1,35 @@
+black==23.9.1
+blinker==1.6.2
+certifi==2023.7.22
+charset-normalizer==3.2.0
+click==8.1.7
+colorama==0.4.6
+Flask==2.3.3
+idna==3.4
+itsdangerous==2.1.2
+Jinja2==3.1.2
+MarkupSafe==2.1.3
+mypy==1.5.1
+mypy-extensions==1.0.0
+node==1.2.1
+npm==0.1.1
+odict==1.9.0
+optional-django==0.1.0
+packaging==23.1
+pathspec==0.11.2
+platformdirs==3.10.0
+plumber==1.7
+pprintpp==0.4.0
+python-dotenv==1.0.0
+requests==2.31.0
+typing_extensions==4.7.1
+urllib3==2.0.4
+Werkzeug==2.3.7
+zope.component==6.0
+zope.deferredimport==5.0
+zope.deprecation==5.0
+zope.event==5.0
+zope.hookable==5.4
+zope.interface==6.0
+zope.lifecycleevent==5.0
+zope.proxy==5.0.0
```

Comment:
Do we need all of these? We only have a few dependencies so far

Reply:
Not entirely sure, I just did pip freeze, we most likely only need dot-env, flask, requests,pprintpp and npm. This is just to be safe.